dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -223,22 +223,7 @@ public static void setRepositories(
       Iterable<String> mavenRepositoryUrls, boolean addMavenCentral) {
     ImmutableList.Builder<RemoteRepository> repositoryListBuilder = ImmutableList.builder();
     for (String mavenRepositoryUrl : mavenRepositoryUrls) {
-      try {
-        // Because the protocol is not an empty string, this URI is absolute.
-        new URI(mavenRepositoryUrl);
```

Comment:
This validation logic is moved to `mavenRepositoryFromUrl` so that the validation logic can be reused in `ClasspathCheckerArguments`.